recipes/tiledb/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "TileDB" %}
+{% set version = "1.2.0" %}
+{% set sha256 = "6e613d222c5f15537dbbd0cf121138c27d480780cc56acd5a7d1ff9f9ce0fe56" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/TileDB-Inc/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  features:
+    - vc10  # [win and py34]
+    - vc14  # [win and py>=35]
+
+requirements:
+  build:
+    - toolchain
+    - cmake >=3.2
+    - zlib
+    - bzip2
+    - zstd
+    - lz4-c 
+    - blosc
```

Comment:
I don't think this package is on conda-forge

Reply:
Maybe I am missing something, but isn't it here?
https://github.com/conda-forge/blosc-feedstock

Reply:
Nope I must have misspelled it in google :sheep: